src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -243,10 +244,27 @@ export const BankInfoCNP = ({
         formData={formData}
         formPrefix={formPrefix}
         formSubmit={saveBankInfo}
-        isSaving={directDepositUiState.isSaving}
-        onClose={closeDDForm}
-        cancelButtonClasses={['va-button-link', 'vads-u-margin-left--1']}
-      />
+      >
+        <LoadingButton
+          aria-label="update your bank information for compensation and pension benefits"
+          type="submit"
+          loadingText="saving bank information"
+          className="usa-button-primary vads-u-margin-top--0 vads-u-width--full small-screen:vads-u-width--auto"
+          isLoading={directDepositUiState.isSaving}
+        >
+          Update
+        </LoadingButton>
+        <button
+          aria-label="cancel updating your bank information for compensation and pension benefits"
+          type="button"
+          disabled={directDepositUiState.isSaving}
+          className="va-button-link vads-u-margin-left--1"
+          onClick={closeDDForm}
+          data-qa="cancel-button"
```

Comment:
I know that this is not new in this PR, but can you remind me why we are using `data-qa` and not `id`?

Reply:
I probably added that a long time ago, before we used Testing Library and its `data-testid` convention. Years ago I heard someone say you shouldn't use `id`s or `class`es when selecting elements in your e2e tests. They advocated adding a `data-qa` attribute and using that to select elements in your tests. We can and should delete this now as I also deleted the old tests that used it. But I don't want to risk upsetting Jenkins so I'll do that in another PR :(

Reply:
Makes sense, thanks for clarifying! 👍